prime-dotnet-webapi/Models/SiteRegistration/RemoteUser.cs

```diff
@@ -30,6 +30,6 @@ public class RemoteUser : BaseAuditable
         /// </summary>
         public bool Notified { get; set; }
 
-        public ICollection<RemoteUserCertification> RemoteUserCertifications { get; set; }
+        public RemoteUserCertification RemoteUserCertification { get; set; }
```

Comment:
have you checked with the prod data to see if any existing users have multiple certifications before making this change?

Reply:
Yes. There are no remote users with multiple certifications